easybuild/easyconfigs/m/matplotlib/matplotlib-2.1.1-intel-2017b-Python-2.7.14.eb

```diff
@@ -0,0 +1,46 @@
+easyblock = 'Bundle'
+
+name = 'matplotlib'
+version = '2.1.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://matplotlib.org'
+description = """matplotlib is a python 2D plotting library which produces publication quality figures in a variety of
+ hardcopy formats and interactive environments across platforms. matplotlib can be used in python scripts, the python
+ and ipython shell, web application servers, and six graphical user interface toolkits."""
+
+toolchain = {'name': 'intel', 'version': '2017b'}
+
+# this is a bundle of Python packages
+exts_defaultclass = 'PythonPackage'
+
+dependencies = [
+    ('Python', '2.7.14'),
+    ('libpng', '1.6.32'),
+    ('freetype', '2.8'),
```

Comment:
@boegel - my only query here is why the `freetype` version is lower for the Python 2 versus Python 3 versions of this easyconfig.

Reply:
Nice catch. I should lower `freetype` to `2.8` as well in `matplotlib-2.1.1-intel-2017b-Python-3.6.3.eb` (which got merged via #5536), to maintain compatibility with various existing easyconfigs including `X11`.